### DISCUSSION TITLE
JBang's and JReleaser's love child

### DISCUSSION THREAD

**Discussion by helpermethod** [jbangdev/jbang#1423] [Show and tell]
This project

https://github.com/helpermethod/p2e

creates a native image from a JBang script (using the new `export native` option), assembles it to a binary distribution using JReleaser, and makes it available via Homebrew (also using JReleaser).

**Comment by quintesse** [comment]
Nice!

Now, to make things perfect, you only need a `jbang-catalog` repository so people can just run `jbang p2e@helpermethod` (in case they don't want to install a simple binary release :-) )

**Reply by helpermethod** [reply]
https://github.com/helpermethod/jbang-catalog
